recipe/install-libintl-devel.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+mkdir -p ${PREFIX}/include
+cp ./gettext-runtime/intl/libintl.h ${PREFIX}/include/libintl.h
+
+pushd ${PREFIX}/lib
+if [[ "${target_platform}" == osx-* ]]; then
+  ln -s libintl.*.dylib libintl.dylib
+else
+  ln -s libintl.so.* libintl.so
```

Comment:
Other scripts hardcode checking for the particular library version (e.g., `.so.0`) — should this script do likewise?